contracts/contracts/strategies/aerodrome/AerodromeAMOStrategy.sol

```diff
@@ -0,0 +1,860 @@
+                    token1: OETHb,
+                    tickSpacing: 1,
+                    tickLower: lowerTick,
+                    tickUpper: upperTick,
+                    amount0Desired: WETHBalance,
+                    amount1Desired: OETHbRequired,
+                    // we don't need slippage protection checking for that in _checkLiquidityWithinExpectedShare
+                    amount0Min: 0,
+                    // we don't need slippage protection checking for that in _checkLiquidityWithinExpectedShare
+                    amount1Min: 0,
+                    recipient: address(this),
+                    deadline: block.timestamp,
+                    // needs to be 0 because the pool is already created
+                    // non zero amount attempts to create a new instance of the pool
+                    sqrtPriceX96: 0
+                })
+            );
+
+            tokenId = mintedTokenId;
+
+            updateUnderlyingAssets();
+            emit LiquidityAdded(
+                WETHBalance, // WETHAmountDesired
+                OETHbRequired, // OETHbAmountDesired
+                WETHAmountSupplied, // WETHAmountSupplied
+                OETHbAmountSupplied, // OETHbAmountSupplied
+                mintedTokenId, // tokenId
+                underlyingAssets
+            );
+        } else {
+
+            (,uint256 WETHAmountSupplied, uint256 OETHbAmountSupplied) = positionManager.increaseLiquidity(
+                INonfungiblePositionManager.IncreaseLiquidityParams({
+                    tokenId: tokenId,
+                    amount0Desired: WETHBalance,
+                    amount1Desired: OETHbRequired,
+                    // we don't need slippage protection checking for that in _checkLiquidityWithinExpectedShare
+                    amount0Min: 0,
+                    // we don't need slippage protection checking for that in _checkLiquidityWithinExpectedShare
+                    amount1Min: 0,
+                    deadline: block.timestamp
+                })
+            );
+
+            updateUnderlyingAssets();
+            emit LiquidityAdded(
+                WETHBalance, // WETHAmountDesired
+                OETHbRequired, // OETHbAmountDesired
+                WETHAmountSupplied, // WETHAmountSupplied
+                OETHbAmountSupplied, // OETHbAmountSupplied
+                tokenId, // tokenId
+                underlyingAssets
+            );
+        }
+
+        // burn remaining OETHb
+        _burnOETHbOnTheContract();
+        positionManager.approve(address(clGauge), tokenId);
+        clGauge.deposit(tokenId);
+    }
+
+    /**
+     * @dev Check that the liquidity in the pool is withing the expected WETH to OETHb ratio
+     */
+    function _checkLiquidityWithinExpectedShare() internal {
+        (uint256 WETHPositionBalance, uint256 OETHbPositionBalance) = getPositionPrincipal();
+        require(WETHPositionBalance + OETHbPositionBalance > 0, "Can not withdraw full position");
```

Comment:
Is this the right error message for this require?

Reply:
changed to `No liquidity in position`